main.tf

```diff
@@ -0,0 +1,92 @@
+# ----------------------------------------------------------------------------------------------------------------------
+# Terraform and AWS Provider blocks
+# ----------------------------------------------------------------------------------------------------------------------
+terraform {
+  # This module is now only being tested with Terraform 0.13.x.
+  required_version = ">= 0.13.0"
+}
+
+provider "aws" {
```

Comment:
Providers should not be created inside a reusable module and its creation is delegated to the caller module